src/ergw_aaa_ro.erl

```diff
@@ -323,7 +323,7 @@ handle_cca(['CCA' | #{'Result-Code' := Code} = Avps],
     {ok, Session, Events, State};
 handle_cca([Answer | #{'Result-Code' := Code}], Session, Events, _Opts, State)
   when Answer =:= 'CCA'; Answer =:= 'answer-message' ->
-    {{fail, Code}, Session, [stop | Events], State};
+    {{fail, Code}, Session, [{stop, Answer} | Events], State};
```

Comment:
Knowing which procedure cause the stop is helpful, but the failure code would also make sense.